### ISSUE TITLE
Please Consider Changing Name of the Package or Add some relevant description.

### ISSUE THREAD

**Issue by High-Flyer** [emacs-vs/line-reminder#31]
Greetings,
This is the package I was looking, Switched from Notepad++ And was trying to Make emacs looks exactly the same. No other packages can match this, some version control packages falsely named as such, They are not interactive, or really meant for the Purpose.
Or use something from here: [Exact Problem for what the 'line-reminder' is made](https://stackoverflow.com/questions/77631543/emacs-how-to-highlight-unsaved-and-saved-recent-changes-in-the-opened-buffer) for what the 'line-reminder' is made for

Description can be following:
>  Highlight unsaved and saved recent changes in the opened buffer.
>  Highlight buffer modifications
>  Margin Changes Highlight
Very nice packages, Gracias and warm regards.

**Comment by jcs090218** [comment]
Yeah, this is one of the earliest packages I made in 2018. I regretted the name of this package, but it has been with this name for this long, and I didn't come up with a better name. Therefore, I haven't made any move regarding the package name.

> Description can be following:

Feel free to open a PR for this! Changing the description should be the best move at the moment. :)

**Comment by High-Flyer** [comment]
Gimme some time, I am learning how to do PR (My best guess is Pull Request) and commit messages :P.
Unless you wanna teach me, I will try my best to learn and offer those changes in Description.
Thank you again
Regards.

**Comment by chookity-pokk** [comment]
Hey @High-Flyer, if you need some help learning how to use git, I can give you a hand.

**Comment by siwei-li** [comment]
I also would like to thank the author for this package, it looks so nice!
